### PR TITLE
Populate <head />

### LINKED ISSUE #425: Project-specific Open Graph Metadata Tags for Social Media Linking+Sharing
This issue was originally posted to Panoptes-Front-End, but on @camallen's suggestion I've moved it here as a potential feature that would be great to have.

The template HTML for Panoptes projects constructed with the Project Builder all use the same general Zooniverse Open Graph (og) tags that are used by Facebook and Twitter to construct nice looking images+links on their timelines / feeds.

## Current behavior

The default Zooniverse values for these tags are pretty uninteresting, leading to non-specific visuals for links to projects (here, for [Local Group Cluster Search](https://www.zooniverse.org/projects/lcjohnso/local-group-cluster-search)):

`<meta property="og:type" content="website" />
<meta property="og:title" content="Zooniverse" />
<meta property="og:description" content="The Zooniverse is the world’s largest and most popular platform for people-powered research." />`
 
twittercard_old

## Proposed behavior

Project-specific Customized tags: 
1. og:title = Project Name
2. og:description = Project Description
3. og:image = Project Avatar
4. Optional -- og:image:width & og:image:height = sensible size for avatar (200x200)

Here's an example created automatically by WordPress for a Planet Four blog post, but the specifics here are just used to show an example of an attractive card: 
`<meta property="og:title" content="6 Earth Years of Planet Four" /><meta property="og:description" content="Happy Birthday Planet Four. This month marks 6 years of Planet Four. We couldn&#8217;t do any of this without the Planet Four volunteer community.  Thank you for all of your help and contributions.…" /><meta property="og:image" content="http://1.gravatar.com/blavatar/37f000fbf9d4d38c9fb242a24af25556?s=200&amp;ts=1549308161" /><meta property="og:image:width" content="200" /><meta property="og:image:height" content="200" />`

twittercard_blog

### Relevant Links
https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup
https://developers.facebook.com/docs/sharing/webmasters/

### PR DESCRIPTION
**Pull request by rogerhutchings**
Package: app-project

Fills out title, description and other meta tags.

Closes #425 .

TODO:

- [ ] Pull in the origin from an environment variable in order to populate the URL correctly. (Created as #468, since I want to have a look into the correct way to do this.)
- [ ] Ultimately make this reusable and move to `@zooniverse/react-components`

### CODE REVIEW

File: packages/app-project/components/Head/HeadContainer.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    const host = 'http://localhost:3000'
+    const { slug } = this.props.project
+    return `${host}/projects/${slug}`
+  }
+
+  render () {
+    return (
+      <Head
+        description={this.getProjectDescription()}
+        ogImage={this.getProjectBackgroundImage()}
```
Comment (eatyourgreens) [inline packages/app-project/components/Head/HeadContainer.js]: Should this be the project avatar?

Comment (rogerhutchings) [reply packages/app-project/components/Head/HeadContainer.js]: I thought the background is likely to be a large image, and lend itself to `summary-card-with-large-image`: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image

Comment (eatyourgreens) [reply packages/app-project/components/Head/HeadContainer.js]: #425 says use the avatar, so it's worth checking this with project owners.

Comment (rogerhutchings) [reply packages/app-project/components/Head/HeadContainer.js]: Avatar it is then, with background as a fallback

Comment (eatyourgreens) [reply packages/app-project/components/Head/HeadContainer.js]: 👍 In theory, you can't launch a project without having both, so all the live projects should have them.

Comment (rogerhutchings) [reply packages/app-project/components/Head/HeadContainer.js]: Cool, this should be ready to go now

**Pull request merged**